src/blocks/helpers/block-utility.js

```diff
@@ -147,7 +159,7 @@ export const addBlockId = ( args ) => {
 	 *
 	 * @type {Array.<string>}
 	 */
-	const blockIDs = window.themeisleGutenberg.blockIDs;
+	const blockIDs = window.themeisleGutenberg.blockIDs || [];
 
 	if ( attributes === undefined || setAttributes === undefined ) {
 		return ( savedId ) => {
```

Comment:
@HardeepAsrani, for some unknown reasons, when I insert a Leaflet block, all the cleaning functions of the use effect Hook start looping for all the blocks in the reusable. If you insert `debugger;` here, then add a Leaflet block, the cycle will begin.

Reply:
Or `console.count("clean");`

Reply:
https://user-images.githubusercontent.com/17597852/163117640-8c814aa1-1761-4ac8-8e62-a69450fec337.mp4